dist/hls.js

```diff
@@ -9051,6 +9051,29 @@ function () {
       end: bufferEnd,
       nextStart: bufferStartNext
     };
+  } // Trying to find buffered position within `maxFragLookUpTolerance`. Returns same position if
+  // current time is already inside of buffered range or there is no buffered fragment
+  // within `maxFragLookUpTolerance`.
+  ;
```

Comment:
```suggestion
  }; 
  // Trying to find buffered position within `maxFragLookUpTolerance`. Returns same position if
  // current time is already inside of buffered range or there is no buffered fragment
  // within `maxFragLookUpTolerance`.
```

Reply:
@thomasweld it's a dist folder which is automatically generated by running `npm run build`, no need to change anything here